app/controllers/lti_nrps_controller.rb

```diff
@@ -64,11 +64,13 @@ def request_access_token
   end
 
   # NRPS endpoint for Autolab to send an NRPS request to LTI Advantage Platform
-  action_auth_level :send_nrps_request, :instructor
-  def send_nrps_request
+  action_auth_level :sync_roster, :instructor
+  def sync_roster
+    params.require(:lcd_id)
+
     lcd = LtiCourseDatum.find(params[:lcd_id])
     if lcd.nil? || lcd.membership_url.nil? || lcd.course_id.nil?
-      raise LtiLaunchController::LtiError.new("Unable to update roster", :bad_request)
```

Comment:
This seems to be undoing Joey's change, should be reverted (Leave the `LtiLaunchController::` in front)

Reply:
Will do